README.md

```diff
@@ -66,7 +66,60 @@ By definition, members work on their respective squad, although they are free to
 
 ## 4. Release process
 
-// Work in progress ⚠️
+### Release engineer as a role
```

Comment:
I'd consider putting it into a separate file in subfolder, i.e. `Release Process/README.md`. This way when opening this folder on GitHub it will show this readme as if it was the main readme. And we can link to it from the main Readme (I think it also supports sections for headers out of the box)

Reply:
Let's do that after we have these different `Work in progress` fixed. Right now I think we should focus on the standard (what sort of information we share) and the format of the information. We can look at the location later.